cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -0,0 +1,205 @@
+import pandas as pd
+import numpy
+import anndata
+import os
+
+SCHEMA_VERSION = "2.0.0"
+FIXTURES_ROOT = os.path.join(os.path.dirname(__file__))
+
+# Pre-made example files
+h5ad_dir = os.path.join(FIXTURES_ROOT, "h5ads")
+h5ad_valid = os.path.join(h5ad_dir, "example_valid.h5ad")
+h5ad_invalid = [
+    "example_invalid_CL.h5ad",
+    "example_invalid_assay.h5ad",
+    "example_invalid_disease.h5ad",
+    "example_invalid_organism.h5ad",
+    "example_invalid_primary_data.h5ad",
+    "example_invalid_sex.h5ad",
+    "example_invalid_tissue.h5ad",
+    "example_invalid_ethnicity.h5ad",
+    "example_invalid_development_stage.h5ad",
+    "example_invalid_uns.h5ad",
+]
```

Comment:
As a test reader, it's opaque as to what the actual invalid data item is, other than from the name. So from the standpoint of verifying that a test is accurate, I'm wondering if this fixture-based test setup strategy could be replaced by a more dynamic one: E.g., starting with a single _valid_ AnnData object--which is constructed programmatically so the reader can easily see its contents in the test code--and then for each error test case, mutating the one item that should cause a validation to be triggered.